synctl

```diff
@@ -162,10 +162,31 @@ synctl patch test <id> --frequency 5"""
 UPDATE_USAGE = """synctl update {test,alert} <id> [options]
 
 examples:
-# update Synthetic test
-synctl get test <id>  --show-json > test.json
-edit test.json
-synctl update test <id> --file/-f test.json
+# update synthetic test/smart alert using file
+synctl get test/alert <id>  --show-json > <file-name>.json
+edit <file-name>.json
+synctl update test/alert <id> --file/-f <file-name>.json
+
+# update test with multiple options
+synctl update test <test-id> --label "simple-ping" \
+    --description "this is a test example" \
+    --url "https://www.ibm.com" \
+    --location "$LOCATION1" "$LOCATION2" "$LOCATION3" ... \
+    --retries 2 \
+    --retry-interval 2 \
+    --follow-redirect true \
+    --timeout 1m \
+    --allow-insecure true \
+    --frequency 5 \
+    --app-id "$APP-ID" \
```

Comment:
only one `\` this convert the command to a very long line. use double `\\` here.